src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -103,9 +103,8 @@ The .NET Foundation licenses this file to you under the MIT license.
   </PropertyGroup>
 
   <ItemGroup Condition="'$(NativeCodeGen)' == 'llvm'">
-    <!-- TODO-LLVM: remove the IL objects when the IL backend is gone -->
     <LlvmObjectNames Include="$(TargetName)" />
-    <LlvmObjectNames Include="$(TargetName)clrjit" />
+    <LlvmObjectNames Include="$(TargetName).data" />
     <LlvmObjectNames Include="$(TargetName).external" />
```

Comment:
Notable change: formerly the bitcode file produced by ILC (with data symbols) was the "main" one, while all the code was in `clrjit.bc`. Now the file with code is the "main" one and ILC-produced bitcode is named `.data.bc`.